packages/ember-metal/lib/mixin.js

```diff
@@ -257,8 +257,12 @@ function mergeMixins(mixins, m, descs, values, base, keys) {
 
 let IS_BINDING = /^.+Binding$/;
```

Comment:
I ended up removing the regexp and the export. There are only 208 results on GH search, and I reviewed about 50% of them are forks and gists of ember-metal/mixin itself.